engarde/checks.py

```diff
@@ -246,7 +247,30 @@ def one_to_many(df, unitcol, manycol):
     return df
 
 
-__all__ = ['is_monotonic', 'is_shape', 'none_missing', 'unique_index', 'within_n_std',
-           'within_range', 'within_set', 'has_dtypes',
-           'verify', 'verify_all', 'verify_any']
+def is_same_as(df, df_to_compare, **kwargs):
+    """
+    Assert that two pandas dataframes are the equal
+
+    Parameters
+    ==========
+    df : pandas DataFrame
+    df_to_compare : pandas DataFrame
+    **kwargs : dict
+        keyword arguments passed through to panda's ``assert_frame_equal``
+
+    Returns
+    =======
+    df : DataFrame
+
+    """
+    try:
+        tm.assert_frame_equal(df, df_to_compare)
```

Comment:
You'll have to pass through `**kwargs` here to `tm.assert_frame_equal` :) This lets you do things like

```python
df.pipe(is_same_as, other, check_names=False)
```